src/gui/torrentcontentmodel.cpp

```diff
@@ -91,14 +91,18 @@ namespace
         QIcon icon(const QFileInfo &info) const override
         {
             SHFILEINFO sfi = { 0 };
-            HRESULT hr = ::SHGetFileInfoW(info.absoluteFilePath().toStdWString().c_str(),
+            QString nativePath = Utils::Fs::toNativePath(info.absoluteFilePath());
```

Comment:
Just checked. Extension indeed works. But I think the code is a bit ugly:
```c++
HRESULT hr = ::SHGetFileInfoW(QString("." + info.suffix()).toStdWString().c_str(),
```

I prefer:
```c++
HRESULT hr = ::SHGetFileInfoW(info.fileName().toStdWString().c_str(),
```

What do you think?